src/redux/features/appVersion/appVersionSelectors.ts

```diff
@@ -0,0 +1,14 @@
+import { RootState } from 'src/redux/rootReducer';
+import { createSelector } from '@reduxjs/toolkit';
+
+export const selectAppVersion = (state: RootState) => state.appVersion.version;
+const currentAppVersion = process.env.REACT_APP_TERRAWARE_FE_BUILD_VERSION;
+
+export const selectIsAppVersionStale = (state: RootState) =>
+  !!state.appVersion.version && state.appVersion.version.toString().trim() !== currentAppVersion;
+
+// simple example of composable selectors
+export const EXAMPLE_selectIsAppVersionStale_EXAMPLE = createSelector(
+  selectAppVersion,
+  (appVersion) => appVersion && appVersion.toString().trim() !== currentAppVersion
+);
```

Comment:
this is an example of how selectors can be chained and composed together. In this case its unnecessary, but it can be imagined how composing multiple selectors can produce powerful chains of results